samples/extensions/open_cl_interop/open_cl_interop.cpp

```diff
@@ -613,9 +613,9 @@ void OpenCLInterop::prepare_open_cl_resources()
 
 	cl_data->command_queue = clCreateCommandQueue(cl_data->context, cl_data->device_id, 0, &result);
 
-	auto   kernel_source      = vkb::fs::read_shader("open_cl_interop/procedural_texture.cl");
-	auto   kernel_source_data = kernel_source.data();
-	size_t kernel_source_size = kernel_source.size();
+	auto        kernel_source      = vkb::fs::read_shader("open_cl_interop/procedural_texture.cl");

```

Comment:
was this change intentional? This could possibly cause merge conflicts with my OpenCL PR, as I had to rename and move this arm specific sample.

Reply:
Will need to look into this further but this PR should not need to change samples. Likely a rebase issue

Reply:
Fixed